### ISSUE TITLE
I have hosting

### ISSUE THREAD

**Issue by yelluw** [Lambda-Cartel/everything#12]
Wanted to let you know that pay for web hosting on webfaction.com

Its not super powerful and its Linux only (no windows support), but its fairly good and already paid for. We can deploy stuff there.
@bizmonger @awright18 @BryanWilhite

**Comment by bizmonger** [comment]
I think we need Windows for F# backend...




Get Outlook for Android







On Thu, May 25, 2017 at 10:42 PM -0400, "Pablo Rivera" <notifications@github.com> wrote:












Wanted to let you know that pay for web hosting on webfaction.com


Its not super powerful and its Linux only (no windows support), but its fairly good and already paid for. We can deploy stuff there.

@bizmonger @awright18 @BryanWilhite

**Comment by awright18** [comment]
We can use F# on .NET core.

**Comment by BryanWilhite** [comment]
Did anyone reach out to Ody?

Is this webfaction deal just a server?

**Comment by yelluw** [comment]
Its a web hosting plan I pay for. Not a vps but a shared host. Low end but might be useful for something. Just letting you all know in case.

**Comment by odytrice** [comment]
@awright18 Yeah thought about the same thing. I was thinking of using F# with .Net Core so that we could host it anywhere. I personally would prefer we use a Cloud Provider like AWS or Azure. I would like to hear you guys thoughts
@BryanWilhite Thanks for the mention.

**Comment by awright18** [comment]
Pros of AZURE
* We can get started for FREE, with free Azure websites using an *.azurewebsites.net address
* Its world scale, can put our content closer to the users
* Visual Studio Team Services - the free online build and deploy server works well with it
* A ton of platform options if we ever need them. 
* Simple to share ownership of administration
* Easy to set up HTTPS for free

CONS
* @yelluw  isn't a Microsoft DEV or FAN
* It will eventually cost a minimal amount of money for hosting with a domain or other services.  - (I don't mind funding this.)
* Have to pick a starting data center

**Comment by yelluw** [comment]
I'm not against any platform. :)
Not a big windows user (only installed it to run VS with F#). But that's OK. I support the choice you all make.

Azure is OK ,and it's something I do have experience with (along with Amazon). Both of their dashboards and UI suck and it will be a good thing to learn them. ;)

We can use my hosting if needed.

**Comment by yelluw** [comment]
@awright18 @odytrice Let's go with Azure. No sense in discussing this too much. :)

**Comment by BryanWilhite** [comment]
great this worked out, I will start to set up Azure on 5/30 (hope this will be fast enough) all of you (@yellow, @bizmonger, @awright18, @odytrice) will have admin permissions---based on Azure conventions we will all log into lambdacartel.visualstudio.com ---this team site can hook up our GitHub repo with one of ten free web sites (apps).

**Comment by yelluw** [comment]
Thats awesome @BryanWilhite

Are there any online tutorial/guides/docs you recommend?

**Comment by BryanWilhite** [comment]
yes, @yelluw, I will be learning this right along with you: https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-continuous-delivery-use-vso

**Comment by yelluw** [comment]
Those are great docs. Is there an option to those who use visual studio code?

**Comment by BryanWilhite** [comment]
Hey @yelluw, I haven’t read this one yet: “[Use Azure portal to setup Continuous Delivery for Web App On Linux](https://blogs.msdn.microsoft.com/visualstudioalm/2017/05/10/use-azure-portal-to-setup-continuous-delivery-for-web-app-on-linux/)”

**Comment by awright18** [comment]
I think @odytrice May have set up something in VSTS, but I setup a Lambda Cartel Subscription in Azure that I'd be happy to add people to.  I'd prefer one or two primary "Contributors" who can basically do anything, maybe those who care about the build/deploy @odytrice  & @BryanWilhite, but I can also add people with more granular permissions if needed.  I volunteered to pay the costs for hosting the actual project with a URL so that is why I had set up the subscription.   I think @odytrice  has the domain name and will likely have to add some DNS records for azure websites to work correctly.   

I think all I need is the Microsoft account email names for whomever wants to run this stuff.  I'd prefer a twitter DM of things like that unless you don't care about your email being on this open source forum.

**Comment by BryanWilhite** [comment]
Nice move @awright18 :) however, @yelluw might point out that Lambda Cartel might want to do everything as **free** and open source as possible---which might include that our hosting should be free. I recognize that this is a profitable weakness with Azure that I thought @awright18 could avoid. (I am almost sure that Azure uses credit cards to ID people as a second factor---I thought this used to be optional.)

We do not want to make a mistake at @awright18's expense that might be a fifty extra bucks that month. Something as innocent as leaving a powerful virtual machine on for a day could cost about 10 bucks. I think we might want to consider accidentally costing @awright18 money.

**Comment by awright18** [comment]
The biggest issue with azure is they won't let you use app services for example with a domain for free. I think we would be paying for that anywhere.  I can make granular permissions, but I'd have to be in charge of setting up the actual app service and then granting permissions on that particular app service to be able to access the credentials for deployments or whatever, That is fine if that is how we want to do it.   I just don't think we are going to look very cool if we have a website that is like lamdacartel.azurewebsites.net or something to that effect however, I'm ok if we start out that way.

**Comment by awright18** [comment]
Now that I think about it, I'm starting to lean towards another host, because of the cost of what we need. To get even SSL you have a have dedicated box on Azure.  If we just want shared hosting and don't think we care about SSL its about $13 a month to use a domain. That's not a big deal.  Jumping to a dedicated machine with SSL is about $55 a month which is probably over kill for starting out. I think we could probably get that cheaper elsewhere if we had the need.  Maybe we should start with either @yelluw's idea and grow from there if needed.  We can still use VSTS or AppVeyor whatever.  Sorry for running in circles on this.

**Comment by awright18** [comment]
One other thing I just realized...Cloud Flare is free which provides free SSL, and free CDN, and a lot of other security type things for free.  So any hosting plan including $13 per month Azure plan or @yelluw's option would work just fine.  Again sorry for the circles.

**Comment by BryanWilhite** [comment]
I am moving https://github.com/Lambda-Cartel/Nikeza/issues/1 to here...

**Comment by BryanWilhite** [comment]
I am closing this issue and moving it to https://github.com/Lambda-Cartel/everything/issues/12

**Comment by BryanWilhite** [comment]
I think we should go with @yelluw's original idea with the intention of leveraging [GitHub webhooks](https://help.github.com/articles/about-webhooks/). This StackOverflow article shows that I have a lot to learn about this: https://stackoverflow.com/questions/17367247/pushing-from-github-to-a-web-server
